packages/cart/src/services/cart-module.ts

```diff
@@ -636,7 +636,7 @@ export default class CartModuleService<
         ? methodIdsOrSelector
         : [methodIdsOrSelector]
     }
-    await this.shippingMethodService_.delete(toDelete, sharedContext)
+    await this.shippingMethodService_.softDelete(toDelete, sharedContext)
```

Comment:
question: Shouldn't we just use the abstract soft delete implementation that does that? also, in that case, we would have to remove all those implementation and rename the service interface methods

Reply:
I think soft remove is more appropriate for ann informtion POV but also because in other places you might have remove which then collides in behavioural with other modules

Reply:
I'll do this in a follow-up PR to not increase the scope more than it already has.